### ISSUE TITLE
Has this project dead?

### ISSUE THREAD

**Issue by diandianzz** [rttrorg/rttr#363]
Has this project dead?

**Comment by btgoodwin** [comment]
Hard to tell.  My team has made use of it by wrapping it with a serialize/de-serialize functions since that's largely what we need it for various integration services like using Socket.IO Client C++.

[LLDC Reflection](https://github.com/blinemedical/lldc-reflection)

Obviously we're not patching anything big about RTTR in there, though there is a patch as part of a Meson wrap that helps us build in Windows when necessary.  I've been meaning to submit a PR for that specific change, but it does look like no one is really responding anymore here.

**Comment by charlietsao** [comment]
Perhabs one reason is reflection will be added to CXX in the distant future.

**Comment by BlackMATov** [comment]
> Perhabs one reason is reflection will be added to CXX in the distant future.

yes, static reflection will be added to the standard, but this is a little different from the dynamic one that this library is.
